src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/caseofficer/CaseOfficerListCasePersonalisation.java

```diff
@@ -85,6 +101,7 @@ public String getReferenceId(Long caseId) {
                 .put("hearingDate", dateTimeExtractor.extractHearingDate(hearingDateTime))
                 .put("hearingTime", dateTimeExtractor.extractHearingTime(hearingDateTime))
                 .put("hearingCentreAddress", hearingCentreAddress)
+                .put("remoteVideoCallTribunalResponse", asylumCase.read(REMOTE_VIDEO_CALL_TRIBUNAL_RESPONSE, String.class).orElse(""))
```

Comment:
I think this should ideally be placed added in the `PersonalisationProvider` class as it is to do with hearing requirements. See the `buildHearingRequirementsFields(asylumCase, caseListingValues)` method.

Reply:
PersonalisationProvider is not used in this class because the notification template for TCW is very minimal